src/dom-updates.js

```diff
@@ -1,44 +1,57 @@
 import {
   getBookings,
-  getTotalCost
+  getTotalCost,
+  searchByDate
 } from './bookings'
 
 import {
   yourBookings,
-  yourBookingsCost
+  yourBookingsCost,
+  dateField,
+  dashboard,
+  searchResults,
+  availableRooms
 } from './scripts'
 
 const setDashboard = (customer, allBookings, allRooms) => {
+  flatpickr(dateField, {
+    minDate: 'today',
+    altInput: true,
+    altFormat: "F j, Y",
+    dateFormat: "Y/m/d"
+  })
+  
   const usersBookings = getBookings(allBookings, customer);
-  const spent = getTotalCost(allRooms, usersBookings);
-
-  yourBookingsCost.innerHTML = '';
-  yourBookings.innerHTML = '';
-
-  yourBookingsCost.innerHTML = `$${spent}`
+  
+  if (typeof usersBookings === 'string') {
+    yourBookingsCost.innerHTML = `<p>${usersBookings}</p>`
+  } else {
+    const spent = getTotalCost(allRooms, usersBookings);
 
-  usersBookings.forEach(booking => {
-    let room = allRooms.find(room => room.number === booking.roomNumber);
-    let image = setRoomImage(room);
-    let bidet = checkForBidet(room);
+    yourBookingsCost.innerHTML = '';
+    yourBookings.innerHTML = '';
 
-    yourBookings.innerHTML += 
-    `
-    <article class="room">
-      <h3 class="room-type">${room.roomType}</h3>  
-      <p class="booking-date">${booking.date}</p>
-      <img class="room-image" src=${image} alt="photo of a ${room.roomType}">
-      <div class="room-details">
-        <span class="num-beds-container">
-          <p class="num-beds">${room.numBeds}</p>
-          <span class="material-icons-round">bed</span>
-        </span>
-        <p class="bed-size">${room.bedSize}</p>
-        <p class="bidet">${bidet}</p>
-      </div>
-    </article>
-    `
-  })
+    yourBookingsCost.innerHTML = `$${spent}`
+    
+    usersBookings.forEach(booking => {
+      let room = allRooms.find(room => room.number === booking.roomNumber);
+      let image = setRoomImage(room);
+      let bidet = checkForBidet(room);
+  
```

Comment:
You may be able to bundle all of this logic up into a function that gets all the room info and returns these in an object for you to use in your HTML below, just for SRP

Reply:
Beautiful solution. Done.